stable/ark-server/templates/rbac.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: ark
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    app: {{ template "ark-server.name" . }}
+subjects:
+  - kind: ServiceAccount
+    namespace: {{ .Values.namespace.server.name }}
+    name: {{ template "ark-server.sa" . }}
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: rbac.authorization.k8s.io
+{{- end }}
+
+{{- if .Values.rbac.create }}
```

Comment:
Why do you have 2 ifs?

Reply:
you are right, it's a mistake, i'll change it